mysql-test/suite/galera_3nodes_sr/t/galera_sr_threeway_split_no_primary.test

```diff
@@ -0,0 +1,125 @@
+#
+# This test verifies that an orphaned SR gets cleanup upon cluster
+# reconnection.  Specifically, the case where the cluster goes through
+# a state of no primary components.
+#
```

Comment:
This comment could be a little bit more specific. There are other tests SR tests in which the cluster goes through a state of no primary components (like galera_sr_cc_master). However in those tests the node rejoins  with a different id. Here the cluster partitions, and nodes rejoin with the same id.